tests/ja4-rules-7.0.x-01/test.yaml

```diff
@@ -0,0 +1,35 @@
+requires:
+  min-version: 7.0.6
+  lt-version: 8
+  features:
+    - HAVE_JA4
+
+pcap: ../ja4-rules/input.pcap
+
+args:
+  - -k none
+  - --set outputs.1.eve-log.types.5.tls.ja4=off
```

Comment:
this style of `--set` is too fragile for tests. Lets use a minimal suricata.yaml in the test instead